bucket/mizu-cli.json

```
@@ -0,0 +1,13 @@
+{
+    "version": "33.1",
+    "description": "The API Traffic Viewer for Kubernetes",
+    "homepage": "https://getmizu.io/docs/mizu/mizu-cli",
+    "license": "Apache-2.0",    
```

```suggestion
    "license": "Apache-2.0",
```
The trailing whitespace is causing CI to fail.